### PR TITLE
Adjust agent model accessibility visualizations

### PR DESCRIPTION
**Pull request by atte-wsp**
* Figure sizing
* Quintiles in place of deciles
* Accessibility change as purpose specific
* Remove unnecessary plots

### CODE REVIEW

File: scripts/accessibility/helpers.R [hunk 1]
```diff
@@ -18,7 +18,7 @@ add_inc_group <- function(df) {
   df0 <- df %>%
     filter(!age_group %in% "age_7-17") %>%
     arrange(income) %>%
-    mutate(income_group = ceiling(10 * row_number() / n()))
+    mutate(income_group = ceiling(5 * row_number() / n()))
```
Comment (johpiip) [inline scripts/accessibility/helpers.R:21]: Consider making this `ceiling` line a function. Right now, it is difficult to tell what it does. Also, would `quantile` work here?

**Review by johpiip — APPROVED** [review]

File: scripts/accessibility/tidy_agent_data.R [hunk 2]
```diff
@@ -58,14 +58,21 @@ tours_1 <- tours_1 %>%
 
 # Join tours to agents data ----
 
+res_var <- c(
+  "total_access",
+  "sustainable_access",
+  "gen_cost",
+  "cost"
```
Comment (johpiip) [inline scripts/accessibility/tidy_agent_data.R:65]: `gen_cost` and `cost` are not available in my result data. Are they provided in newer versions of Helmet?

**Review by johpiip — APPROVED** [review]

File: scripts/accessibility/tidy_agent_data.R (hunk 2 above)
Comment (atte-wsp) [reply scripts/accessibility/tidy_agent_data.R:65]: Yep, sorry I forgot to update input data. These files are added in Helmet Pull [323](https://github.com/HSLdevcom/helmet-model-system/pull/323). Uploaded these to Sharepoint data folder.

File: scripts/accessibility/helpers.R (hunk 1 above)
Comment (atte-wsp) [reply scripts/accessibility/helpers.R:21]: Seems that `dplyr` has readily available `ntile` function, which should do what I was looking for.  I always get surprised with the extent of these libraries. :)

Comment (johpiip) [reply scripts/accessibility/helpers.R]: That is new for me too! :)

**Review by johpiip — APPROVED** [review]
Thank you for the changes! Everything seems to be working well! Just `V2030_ve0/tours.txt` is missing from Sharepoint, but I tested with 2018 and MAL 2019 scenarios. You can merge! Thank you for your hard work! 🙂

**Pull request merged**